### PR TITLE
Big bug fixing part 1

### PR DESCRIPTION
**Pull request by dvir001**
Trying to fix some issues.

:cl:
- tweak: The price of the Bocadillo is increased to $23500.
- tweak: The amount of cloth required to craft altars has been increased to 3.

### CODE REVIEW

File: Content.IntegrationTests/Tests/PostMapInitTest.cs [hunk 1]
```diff
@@ -27,42 +27,135 @@ namespace Content.IntegrationTests.Tests
...
+            "/Maps/_NF/Shuttles/Scrap/tide.yml",
+            // Sr
+            "/Maps/_NF/Shuttles/Sr/bottleneck.yml",
+            "/Maps/_NF/Shuttles/Sr/broom.yml",
+            "/Maps/_NF/Shuttles/Sr/mailpod.yml",
+            "/Maps/_NF/Shuttles/Sr/parcel.yml",
+            "/Maps/_NF/Shuttles/Sr/watchdog.yml",
+            // Syndicate
+            "/Maps/_NF/Shuttles/Syndicate/hunter.yml",
+            "/Maps/_NF/Shuttles/Syndicate/infiltrator.yml",
```
Comment (Cheackraze) [inline Content.IntegrationTests/Tests/PostMapInitTest.cs:150]: Dear God no. This needs to be smart and pull the shuttles dynamically, not hard code each one into the test

**Review by Cheackraze — COMMENTED** [review]
There's also a ton of unrelated bug fixing going on, makes it hard to review this and should be split into multiple prs

File: Content.IntegrationTests/Tests/PostMapInitTest.cs (hunk 1 above)
Comment (dvir001) [reply Content.IntegrationTests/Tests/PostMapInitTest.cs:150]: Temp till we pass without fails
After that we can just feed it the folder.

**Comment by dvir001** [comment]
@whatston3 Ok so the integration tests workflow is set to off again and I will restore them back on in another PR,
We should have this merged before to clear issues so we can just continue working on this without having tests fails on open PR's.

**Review by whatston3 — APPROVED** [review]
Reverted a lot of changes, bugfixes here (Delta deep fryer, fixes to triggered debug assertions) are useful.  Gallery looks nice, that's a whole lot of crayons, but seems fine.

Restored the Rosebuds, the golden rose, and restored the GPS to the tables they were in.  Consolidated BaseWallmount and BaseStructureWallmount, works well under test.

Changed maps seem to load fine.  Approving this.

**Pull request merged**